lib/index.d.ts

```diff
@@ -151,6 +152,7 @@ export function decryptMIMEMessage(
     getEncryptedSubject: () => Promise<string>;
     verify: () => Promise<number>;
     errors: () => Promise<Error[] | undefined>;
+    getSignature: () => Promise<OpenPGPSignature>;
 };
```

Comment:
And then personally, I would aim to make this return something similar to the `signatures` of `decryptMessage` - see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/839738b973f370fa4d7e1fdad1fcb7deb528adf9/types/openpgp/index.d.ts#L5221-L5225.